content/develop/develop-images/guidelines.md

```diff
@@ -117,3 +117,65 @@ frequently changed where possible.
 
 For more information about the Docker build cache and how to optimize your
 builds, see [cache management](../../build/cache/_index.md).
+
+## Pin base image versions
+
+Image tags are mutable, meaning a publisher can update a tag to point to a new
+image. This is a useful because it lets publishers update tags to point to
+newer versions of an image. And as an image consumer, it means you
+automatically get the new version when you re-build your image.
+
+For example, if you specify `FROM alpine:3.19` in your Dockerfile, `3.19`
+resolves to the latest patch version for `3.19`.
+
+```dockerfile
+# syntax=docker/dockerfile:1
+FROM alpine:3.19
+```
+
+When a new patch version for `3.19` is released, the tag is updated to point to
```

Comment:
**suggestion (non-blocking)**: show an example of patch versions, e.g:

> At one point in time `3.19` points to `3.19.1` but if you rebuild 3 months after it might now point to `3.19.4`

I think seeing a clear example of this would help deliver the point across